_posts/en/newsletters/2020-05-06-newsletter.md

```diff
@@ -0,0 +1,241 @@
+  list ([1][casatta qr], [2][allen qr]) a request for discussion
+  hopefully leading to standardization of a method for visual
+  communication of [Partially Signed Bitcoin Transactions][topic psbt]
+  (PSBTs) and other potentially large data blobs related to Bitcoin
+  wallet interaction.  See the [previous discussion][qr old] in the
+  Specter DIY repository and [continue the discussion][qr new] in the
+  Airgapped Signing repository.
+
+## Field report: running a Lightning node in an enterprise environment
+
+{% include articles/suredbits-enterprise-ln.md %}
+
+## Recently transcribed talks and conversations
+
+*[Bitcoin Transcripts][] is the home for transcripts of technical
+Bitcoin presentations and discussions. In this new <!-- TODO: remove "new" next month --> monthly feature, we
+highlight a selection of the transcripts from the previous month.*
+
+- **Simplicity---Next-Generation Smart Contracting** Adam Back presented at a
+  Blockstream webinar on Simplicity: a next-generation, low-level
+  alternative to Bitcoin Script focused on provable security and greater
+  expressiveness. Back discussed how Simplicity, if enabled in Bitcoin
+  today, would allow developers to implement new functionality like
+  [SIGHASH_NOINPUT][topic sighash_noinput] without necessarily needing a soft fork. He also
+  displayed a [demo][simplicity demo] showing what you can do with
+  Simplicity today.  ([transcript][simplicity xs], [video][simplicity
+  vid], [slides][simplicity slides])
+
+- **Attacking Bitcoin Core** Amiti Uttarwar presented at LA BitDevs.
+  Uttarwar discussed how changes to Bitcoin’s peer-to-peer layer should
+  be assessed according to five goals: reliability, timeliness,
+  accessibility, privacy, and upgradability. She discussed the dangers of
+  network partitions and [eclipse attacks][topic eclipse attacks]---and then explained why
+  block-relay-only connections and anchor nodes are effective
+  mitigations.  ([transcript][attacking xs], [video][attacking vid])
+
+- **LND v0.10** Laolu Osuntokun, Joost Jager, and Oliver Gugger
+  presented in virtual reality at Reckless VR. Osuntokun presented Tor
+  and RPC enhancements in the latest release of LND plus a new channel
+  feature called [anchor outputs][topic anchor outputs] that addresses
+  the challenge of estimating onchain fees months in advance.
+  Jager discussed the challenges of [multipart payments][topic multipath
+  payments] including the splitting algorithm, what happens when the
+  shards of the payment arrive at different times, and strategies for
+  when a multipart payment fails. Gugger ended by discussing Partially
+  Signed Bitcoin Transaction ([PSBT][topic psbt]) channel funding and
+  the channel abstraction work that made this possible.
+  ([transcript][lnd10 xs], [video][lnd10 vid])
+
+- **Grokking Bitcoin** Kalle Rosenbaum both participated in a [Socratic
+  Seminar][] and gave a presentation at London Bitcoin Devs.  The
```

Comment:
I don't think this link would be very meaningful for readers. I suggest either removing the link entirely and rewording to "developer meetup" or linking to https://bitdevs.org/running-a-great-socratic-seminar/.